test/e2e/specs/wp-likes-spec.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import * as driverManager from '../lib/driver-manager';
+import * as dataHelper from '../lib/data-helper';
+import LoginFlow from '../lib/flows/login-flow';
+import PostAreaComponent from '../lib/pages/frontend/post-area-component';
+import CommentsAreaComponent from '../lib/pages/frontend/comments-area-component';
+import GutenbergEditorComponent from '../lib/gutenberg/gutenberg-editor-component';
+
+const host = dataHelper.getJetpackHost();
+const screenSize = driverManager.currentScreenSize();
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const mochaTimeoutMS = config.get( 'mochaTimeoutMS' );
+const blogPostTitle = dataHelper.randomPhrase();
+const blogPostQuote =
+	'The foolish man seeks happiness in the distance. The wise grows it under his feet.\n— James Oppenheim';
+
+describe( `[${ host }] Likes: (${ screenSize })`, function () {
+	let driver;
+	this.timeout( mochaTimeoutMS );
+
+	before( 'Start browser', async function () {
+		this.timeout( startBrowserTimeoutMS );
+		driver = await driverManager.startBrowser();
+	} );
+
+	describe( 'Like posts and comments @parallel', function () {
+		step( 'Login, create a new post and view it', async function () {
+			this.loginFlow = new LoginFlow( driver, 'louisTestUser' ); // tofix: switch to e.g. gutenbergSimpleSiteUser
+			await this.loginFlow.loginAndStartNewPost( 'c3polikes.blog', true );
```

Comment:
👋 You'll need to add the user credentials and site URL to the `encrypted.enc` config to make this work. You can encrypt and decrypt the config with [these commands](https://github.com/Automattic/wp-calypso/blob/582fe5e9e15f5481f811afb7c3c5a33b81d99ad5/test/e2e/package.json#L18-L19). You'll also need some [Environment Variables](https://github.com/Automattic/wp-calypso/blob/bd94441143c65a15f60fb352271fff79db09fa8e/test/e2e/docs/setup.md#environment-variables) set beforehand.

Reply:
Thanks for taking a look, is it ok to just use the gutenbergSimpleSiteUser for this?

I'm going to be reworking some of this after the PR is merged and I'll create new users then.

Reply:
> is it ok to just use the gutenbergSimpleSiteUser for this?

I think that `gutenbergSimpleSiteUser` would be OK to use if all you need is a Simple site to run the suite against. 👍